### PR TITLE
Add .github/ responsible members to codeowners.

### PR DESCRIPTION
**Pull request by NlightNFotis**
Add the members of the diffblue open-source team, the internal team responsible for maintenance of CBMC to the codeowners file as responsible for the github tooling and CI pipeline.

- [ ] Each commit message has a non-empty body, explaining why the change was made.
- [ ] Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- [ ] The feature or user visible behaviour I have added or modified has been documented in the User Guide in doc/cprover-manual/
- [ ] Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- [ ] My commit message includes data points confirming performance improvements (if claimed).
- [ ] My PR is restricted to a single feature or bugfix.
- [ ] White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

**Review by peterschrammel — APPROVED** [review]

**Review by martin-cs — APPROVED** [review]
Seems fair.  A new line at the end of the file would be nice!

**Review by martin-cs — CHANGES_REQUESTED** [review]
I have problems with the second commit.

1. As far as I am aware "being at Diffblue" is not a requirement for being a CODEOWNER.

2. If you are going to revoke or modify CODEOWNER status based on lack of interest / no longer involved (which is long overdue -- there is at least one CODEOWNER who's github handle has changed) then can this be done evenly and not just for `scripts/`.

3. This should be a separate PR with clear criteria (and notification to the people involved), not added on to a different change.

**Comment by martin-cs** [comment]
Also, it looks like the code coverage is broken.  Please can (ironically enough) one of the maintainers for the CI please look at it?

**Comment by NlightNFotis** [comment]
Hi @martin-cs.

1) You are right - employment at Diffblue is not a prerequisite for being a CODEOWNER. Apologies if my wording made it seem that way. The people removed were added to the codeowners group as part of their employment duties and they no longer seem to be engaged with the project after they left the company. This was the basis of their removal - it seemed the pragmatic approach was not to have PRs wait for approval by people no longer engaged.

2) I agree and this PR will have this commit dropped. But I'm personally very keen to clean up the codeowners file (and other janitorial work), so if you could drop me a line with your notes on that I would be thankful.

3) With regards to the coverage, I will investigate it. Though I'm suspicious of it, as we have seen spurious coverage reports in the past.

Cheers, Fotis.

**Review by martin-cs — APPROVED** [review]
@NlightNFotis thanks for the thoughtful response.  I agree that having PRs blocked on people who are no longer involved is bad.  I have e-mailed you some thoughts about how to go about cleaning up CODEOWNERS.  As that is a separate issue, I have no objections to this PR.

**Pull request merged**
